### PR TITLE
added arrows to show current sort ordering

### PR DESCRIPTION
**Pull request by cuzethan**
@evanugarte

### CODE REVIEW

**Comment by cuzethan** [comment]
Screenshot 2024-07-02 at 11 09 36 AM

**Comment by cuzethan** [comment]
Screenshot 2024-07-02 at 11 17 34 AM
Screenshot 2024-07-02 at 11 17 30 AM
Screenshot 2024-07-02 at 11 17 26 AM
used "times used" column

**Comment by cuzethan** [comment]
Screenshot 2024-07-03 at 11 18 48 AM
Screenshot 2024-07-03 at 11 18 52 AM
Screenshot 2024-07-03 at 11 18 56 AM
@evanugarte works properly for all the other columns as well

File: src/Pages/URLShortener/URLShortener.js [hunk 1]
```diff
@@ -425,7 +425,15 @@ export default function URLShortenerPage(props) {
                         className={`${className}`}
                         key={title}
                       >
-                        <button onClick={() => handleSortUrls(columnName)}>{title}</button>
+                        <div className="flex items-center justify-center">
+                          <button onClick={() => handleSortUrls(columnName)}>{title}</button>
+                          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth="1.5" stroke="currentColor" className={`w-5 h-5 ${(currentSortOrder === 'ASC' && currentSortColumn === columnName) ? `` : 'hidden'}`}>
```
Comment (evanugarte) [inline src/Pages/URLShortener/URLShortener.js]: can this class be a function call instead? would be cleaner like

`className={getSortArrowClass(columnName, 'ASC')}`

the function would basically do the same logic, taking a param for `columnName` and `sortOrder` returning a blank string or `'hidden'`.

**Comment by cuzethan** [comment]
@evanugarte i resolved the check errors

File: src/Pages/URLShortener/URLShortener.js (hunk 1 above)
Comment (cuzethan) [reply src/Pages/URLShortener/URLShortener.js]: > @evanugarte resolved

**Review by evanugarte — APPROVED** [review]

**Review by weslayer — APPROVED** [review]
clark just got ethan’d

**Review by steeevin88 — APPROVED** [review]
lgtm

**Pull request merged**